docs/src/tutorials/tutorial_5.md

```diff
@@ -372,20 +390,37 @@ javis(
     demo,
     [
         BackgroundAction(1:550, ground),
-        Action(1:550,
+        Action(
+            1:550,
             :atom,
             (args...) -> element(),
-            subactions = [ 
+            subactions = [
                 SubAction(101:140, Scaling(1, 12)),
                 SubAction(241:280, Scaling(12, 20)),
                 SubAction(381:420, Scaling(20, 7)),
-                SubAction(521:550, Scaling(7, 1))
-            ]
+                SubAction(521:550, Scaling(7, 1)),
+            ],
+        ),
+        Action(
+            1:100,
+            (args...) -> info_box(value = val(:atom)),
+            subactions = [SubAction(1:30, sineio(), appear(:draw_text))],
```

Comment:
As this is a tutorial you should explain what's going on 😛

Reply:
Done.